samples/workflows/case-downtime/README.md

```diff
@@ -0,0 +1,78 @@
+# Workflow with downtime
+
+_Please, consider this sample as a way to guide you and give you ideas to manage your workflow. 
+Of course, feel free to adjust according your needs._
+
+## What is the problem ?
+
+Assuming the size of your project and the techno, you have different needs to play scripts.
+In this sample, we will consider the case :
+- I need to stop my server to deploy a new version. So I have a downtime during X seconds.
+- I have scripts that can take a long to be played.
+
+Because you have a downtime, you want it to be the smallest as possible.
+
+## What can we do ?
+
+So consider these 3 steps :
+- BEFORE : represents all the time before your deployment.
+- DURING : represents all the downtime during the deployment.
+- AFTER : represents all the time after the deployment.
+
+Now imagine you will have an execution of Datamaintain for each step. So to minimise your downtime you have to 
+think about what can be played during the steps BEFORE and AFTER. You also may consider splitting your script in order 
+to play some parts on different steps.
+
+Basically, some tips :
+- BEFORE : all actions that don't break the current version db requirements
+  - adding a new property on document
+  - adding index
+- DURING : all actions that break the requirements of the previous version and are needed for the new version.
+  - updating the name of a property
+  - update the value of a property (i.e. an enum renamed)
+- AFTER : all actions that don't break the new version db requirements
+  - removing unused property on document
+  - removing unused collection
+  - removing index
+
+Please consider reading or section about best practices.
```

Comment:
Misspell: ```or section``` --> ```our section```. 
Could add a link to that section.

Reply:
This section doesn't exist for now :-)